workflow_tests/utils/utils.ts

```diff
@@ -81,10 +105,19 @@ function createMockStep(name, message, job_id = null, inputs = null, in_envs = n
     return mockStep;
 }
 
-function createStepAssertion(name, isSuccessful = true, expectedOutput = null, jobId = null, message = null, inputs = null, envs = null) {
+function createStepAssertion(
+    name: string,
+    isSuccessful = true,
+    expectedOutput = null,
+    jobId: string | null = null,
+    message: string | null = null,
+    // Replace arrays with records
```

Comment:
Expensify tend to avoid TODOs in the code, I'm fine with it, just a little nit.
```suggestion
    // Replace arrays with records
```

Reply:
However I agree we could refactor this, as it seems weird to have an array here